spec/hedgelog/scrubber_replacement_spec.rb

```diff
@@ -17,6 +17,11 @@
       it { should eq 'password:*****' }
     end
 
+    context 'the input looks like a ruby hash' do
+      let(:input) { '"password"=>"bar"' }
+      it { should eq '"password"=>"*****"'}
+    end
+
```

Comment:
Lets update the tests here.  Use `let(:secret)` with a broad character list omitting just the limitations we know we have now.  Also add a value to the end of each test string that we can insure is still present after the masking `publicvar=foo`. 
```suggestion
    # known limitations are the characters we treat as potentially ending this key/value pair
    # ; & , and \s
    let(:secret) { 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789~`!@#$%^*()_+={}[]|\:"\'<>.?/⚠️' }

    context 'the input looks like params' do
      let(:input) { "password=#{secret}&publicvar=foo" }
      it { should eq 'password=*****&publicvar=foo' }
    end

    context 'the input looks like a key value pair' do
      let(:input) { "password:#{secret};publicvar=foo" }
      it { should eq 'password:*****;publicvar=foo' }
    end

    context 'the input looks like a ruby hash' do
      let(:input) { '{"password"=>"'+secret+'","publicvar"=>"foo"}' }
      it { should eq '{"password"=>"*****","publicvar"=>"foo"}'}
    end
```